apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -170,8 +169,7 @@ cleanup_genesis(Val) ->
     Val.
 
 add_missing_to_genesis_block(#{<<"height">> := 0} = Block) ->
-    GB = aec_blocks:serialize_to_map(
-           aec_block_genesis:genesis_block_as_deserialized_from_network()),
+    GB = aec_blocks:serialize_to_map(aec_block_genesis:genesis_block()),
```

Comment:
So attempting to post block with height 0 attempts to read from disk?

Reply:
Yes, this could be a vector for an attack, I'll modify it to use the genesis block from the chain instead